File: API/src/main/java/com.healthtracker.tracker_api/model/SignUpModel.java
package com.healthtracker.tracker_api.model;

import lombok.Getter;

@Getter
public class SignUpModel {
    private final String username;
    private final String password;
    private final String repeatPassword;

    public SignUpModel(String u, String p, String r) {
        this.username = u;
        this.password = p;
        this.repeatPassword = r;
    }
}

File: MVVM/Models/SignUpModel.cs
namespace HealthTrackerMVVM.Models
{
    public class SignUpModel
    {
        public string Username { get; set; }

        public string Password { get; set; }

        public string RepeatPassword { get; set; }

        public SignUpModel()
        {
            this.Username = string.Empty;
            this.Password = string.Empty;
            this.RepeatPassword = string.Empty;
        }
    }
}
